### ISSUE TITLE
AWS Migration [Day Use Pass] [DUE: October 2025]

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/parks-reso-api#431]
#### Description:
We must migrate from AWS ASEA to LZA by October 31, 2025
Launch day: June 24, 2025.
AWS support countdown: September - October, 2025

Please review info in https://github.com/bcgov/parks-reso-api/issues/406 and in the migration Epic for more deets to help with refinement.

#### Acceptance Criteria:
Given we are a product team using AWS ASEA
When it is time to migrate
Then we 'move' over to the new LZA AWS instance.

#### Development Checklist:

- [x] Create new project sets for LZA in the registry
- [ ] Obtain expense authority approval (eMOU process) (note: depending on value, this will either be Lindsay or Rumon)
- [ ] Provision project sets
- [ ] Replicate workloads in Dev/Test environments
- [ ] Validate Dev/Test deployments
- [ ] Create production workloads in LZA
- [ ] Plan and execute your production switch
- [ ] Validate production workloads
- [ ] Decommission ASEA project sets

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

- [LZA kickoff meeting presentation](https://bcgov-my.sharepoint.com/:p:/r/personal/khurrum_husain_gov_bc_ca/Documents/Microsoft%20Teams%20Chat%20Files/LZA%20kick-off%20draft%20slides.pptx?d=wd7f6c556159a4f288e0f8cb25dade935&csf=1&web=1&e=TPeQCB)

#### Notes:
- DUP winter season: usually mid-December to end of March, with a switch from daily to weekends only after the first week of January 
-

**Definition of Ready**

- [x] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [x] Design / Solution is accepted by Product Owner (if applicable)
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

**Comment by LindsayMacfarlane** [comment]
Followed up with Team Osprey on April 22, 2025 - From our understanding, our 'projects' need to be created by the platform team and until then we are blocked on moving forward. Assumption is that it will happen sometime in FY 2025-26. Added the blocked label to this ticket.

**Comment by LindsayMacfarlane** [comment]
I feel this can wait until early Jan. We should confirm with @MarshallHenson if there is a good time to complete the migration in Jan. I'm also assuming we can start setting everything up, but just not finish the migration?

**Comment by LindsayMacfarlane** [comment]
Closing this ticket as it is no longer needed. Platform Services team will wait to retire AWS-ASEA until we've completed our September 2026 reserve-rec (DUP) release.
